packages/lib/environment/auth.ts

```diff
@@ -2,41 +2,33 @@ import { prisma } from "@formbricks/database";
 import { unstable_cache } from "next/cache";
 
 export const hasUserEnvironmentAccess = async (userId: string, environmentId: string) => {
-  if (!userId) return false;
-  const environment = await prisma.environment.findUnique({
-    where: {
-      id: environmentId,
-    },
-    select: {
-      product: {
+  return await unstable_cache(
```

Comment:
if you return the function directly you can instead of
```
export const myFunc = (userId: string) => { return null }
```
can also write 
```
export const myFunc = (userId: string) => null
```

But we can merge, just wanted to mention this ;-)

Reply:
yeah my bad, will update this next time!